packages/twenty-website/src/content/user-guide/settings.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 title: Settings
-info: Learn how to manage permissions in Twenty.
+info: "Learn how to manage your workspace: permissions, billing, data model and much more."
```

Comment:
@StephanieJoly4 you'll need to add quotes around strings if they contain `:` (column symbol). Otherwise the YAML parser will think it's a new key (like title: or info: )